docs/modules/servers/partials/SubAddressing.adoc

```diff
@@ -0,0 +1,22 @@
+=== SubAddressing
```

Comment:
https://james.staged.apache.org/james-project/3.9.0/servers/distributed/configure/mailets.html

It would be great to have an exemple that can easily get copy pasted

Reply:
should i just add that ?

```
<mailet match="All" class="SubAddressing">
</mailet>
```

Reply:
Yes